### DISCUSSION TITLE
don´t seperate Apps and Programs. Integrate Apps into Programs folders

### DISCUSSION THREAD

**Discussion by no-username42** [Open-Shell/Open-Shell-Menu#1901] [Ideas]
I know the technical differences between Metro Apps and classic Windows programs.
But these are irrelevant for most uninformed users and for those who are.

I just want to find the apps I need at the place I expect them to be and not having to search on different locations. (or even remember that the missing program may be hidden somewhere else.)
Therefore I drag and drop the relevant apps to the desktop what results in a .lnk file, that can be (renamed and) placed in the classic folder structure, where it belongs.
So, this is actually possible.
Sadly Open-Shell doesn´t support this on it´s own.

For the implemenation I would sugguest an additional 'shadow' folder structure (similar to the users and all-users/roaming menu folders), but for the Metro Apps. This way Open-Shell can keep track of all Apps and where the user wants them to go.

If a user cares about the differences between apps and real programs, these can be differentiated via a visual clue. Probably an additional Icon, a different background color or an appended text '(App)' (what I would not like).

**Comment by no-username42** [comment]
It was discussed for the classic-shell (http://classicshell.net/forum/viewtopic.php?f=7&t=5038) but I think the given reasons against this are weak / are refuted by the visual differentiation.

**Comment by ge0rdi** [comment]
There is enhancement request #985 for that already.
